tlspuffin/src/tls/seeds.rs

```diff
@@ -2160,16 +1839,15 @@ pub fn create_corpus() -> Vec<(Trace<TlsQueryMatcher>, &'static str)> {
     corpus!(
         // Full Handshakes
         seed_successful: cfg(feature = "tls13"),
+        seed_successful_with_flights: cfg(feature = "tls13"),
```

Comment:
do not have add redundant traces here, so I guess remove seed_successful.

Reply:
Eventually, rename `seed_successful_with_flights` into `seed_successful`

Reply:
I removed `seed_successful` and renamed `seed_successful_with_flights` into `seed_successful`.  Since `seed_successful` was used in `seed_successful_with_ccs`, I changed `seed_successful_with_ccs` to be a stand-alone trace.